### ISSUE TITLE
Problem adding external .sup and idx/sub

### ISSUE THREAD

**Issue by Viking55** [justdan96/tsMuxer#87]
When adding an external .sup-file the chosen language shows as an eight digit number.
idx/sub are not accepted at all.

**Comment by justdan96** [comment]
Would you be able to provide an example invocation? We tend to like Big Buck Bunny for reproducing issues but some screenshots could also help.

**Comment by Viking55** [comment]
You can see the highlighted sub. I demuxed it and added it back and chose English language as it was. I never went on to mux it though but I expect the player to show the number. I can mux it if you want me to. The idx/sub is not accepted by the previous versions either. It was me remembering wrong. However, maybe it should accept them. No big deal if not though.

![Adding sup](https://user-images.githubusercontent.com/58627166/71202376-a5e3d280-22ce-11ea-9d49-2db9c7ae882b.jpg)

**Comment by Viking55** [comment]
Hm, pressed the wrong button...

**Comment by Viking55** [comment]
Ok, muxed it in with the original embedded sub as well. While the original shows up as "eng" in VLC the external sub's language shows as "231".

**Comment by jcdr428** [comment]
@justdan96 @xavery
- GUI 2.6.12 with tsMuxer 2.6.12 : no problems.
- GUI 2.6.15 with tsMuxer 2.6.12 : bug, lang=eng is changed to number.
So the problem is with the GUI commits. Could you please check whether https://github.com/justdan96/tsMuxer/commit/b2ac11b0b50b8071867588b4e165fafb8981b33e#diff-5f60f56cf36cf63bec7eb6849ebe02b8 is the cause of this new bug ?

**Comment by justdan96** [comment]
There's been a lot of changes since 2.6.12, it could be difficult to find when this bug was introduced

**Comment by xavery** [comment]
I'd gladly test this, but I don't have any `.sup` files to test with - if anybody would be kind enough to share such a file, or point me in the right direction in order to generate/extract one, I'll be on my way.

**Comment by jcdr428** [comment]
@xavery you can demux any pgs subtitle to .sup from a Bluray m2ts (eg BigBuckBunny) with tsMuxer. Then when reintroducing the .sup in tsMuxerGUI and selecting the language, the bug appears.

**Comment by xavery** [comment]
@jcdr428 Thanks! I managed to produce a `.sup` file, but it turned out that this issue was also reproducible with audio tracks and `.srt` files as well.
I submitted a PR that fixes this.

**Comment by xavery** [comment]
This is fixed in the [22 December nightlies](https://bintray.com/justdan96/tsMuxer/tsMuxerGUI-Nightly/2019-12-22--01-10-25#files), do kindly retest and report back if everything's fine.

**Comment by jcdr428** [comment]
@xavery I've tested Windows 32 and 64 with sup, srt, ac3, aac, dts... : the bug is fixed.

**Comment by Viking55** [comment]
Hi and Merry Christmas.
Sorry for late reply. I've tested with 22 Dec release and
as you wrote, sup-, srt- and audio files get the correct language now. 
It's much appreciated what you are doing with TSmuxer. 
I don't have a clue about programming, I just use the software.
Are the 3D plane list and UHD sync issues resolved?
I've included 2 screenshots.
Don't know if this is a bug or not but UHD-iso does not show a box for "Disk label" as BD-iso has.
Also, have a look at "Start mux time". BD-iso usually gets this time, a few times a different one.
Why doesn't it start on zero? Or is it some setting with me? 
Regards
//Lars
This is fixed in the 22 December nightlies, do kindly retest and report back if everything's fine.
